storage/memory/log_storage.go

```diff
@@ -342,6 +342,22 @@ func (t *logTreeTX) GetActiveLogIDs(ctx context.Context) ([]int64, error) {
 	return t.getActiveLogIDs(ctx)
 }
 
+func (t *readOnlyLogTX) GetUnsequencedCounts(ctx context.Context) (map[int64]int64, error) {
+	t.ms.mu.RLock()
+	defer t.ms.mu.RUnlock()
+
+	ret := make(map[int64]int64)
+	for id, tree := range t.ms.trees {
+		tree.mu.RLock()
+		defer tree.mu.RUnlock()
+
+		k := unseqKey(id)
+		q := tree.store.Get(k).(*kv).v.(*list.List)
```

Comment:
nit: what does "q" stand for? 

nit2: `tree.store` would benefit from a comment stating what it contains (ie, the tree items are `kv`s and the `kv`'s value is a `*list.List`). Bonus points if it explains why it's using "unusual" containers, instead of the standard maps and slices.